### DISCUSSION TITLE
Can I use "optional chaining operator" (foo?.bar) in AdonisJs 4.1 ?

### DISCUSSION THREAD

**Discussion by maidiries** [adonisjs/core#2125] [Help] [answered]
Optional chaining operator (new feature ES2020)
https://medium.com/@kashanhaider/optional-chaining-operators-in-javascript-es-2020-update-85676d07da95

Error in Adonis when I use it:
`Unexpected token [...] response?.data [...]`

Running on node version 15.5.1 and Adonis.js 4.1

**Comment by thetutlage** [comment, marked as answer]
Adonis doesn't have anything to do with the syntax of the language. If Node supports it, then we do as well
